source/runtime_gui.cpp

```diff
@@ -362,6 +362,7 @@ void reshade::runtime::load_config_gui(const ini_file &config)
 	config_get("INPUT", "KeyFPS", _fps_key_data);
 	config_get("INPUT", "KeyFrameTime", _frametime_key_data);
 	config_get("INPUT", "InputProcessing", _input_processing_mode);
+        config_get("INPUT", "EscClosesOverlay", _esc_closes_overlay);
```

Comment:
This feels like an option that should be in the `OVERLAY` section, since it's not really a hotkey, more like `[OVERLAY] EscapeToClose=<0,1>`.